modules/ecr/main.tf

```diff
@@ -191,6 +191,15 @@ resource "aws_ecr_repository" "ecr_mifos-phee-ns-web-app" {
   name                 = "payment-hub/phee-ns/${var.environment}-web-app"
   image_tag_mutability = "MUTABLE"
 
+  image_scanning_configuration {
+    scan_on_push = true
+  }
+}
+
+resource "aws_ecr_repository" "ecr_mock_sris" {
+  name                 = "mock-sris/${var.environment}"
```

Comment:
Name should be something like "mock-sris/${var.environment}-mock-sris" or similar IMO

Reply:
I choose a short name `mock-sris/$var.env` because I expect to have only one record in `mock-sris` folder, but `mock-sris/${var.environment}-mock-sris` looks okay as well.